lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1506,8 +1508,9 @@ sub register_assets_from_settings ($self) {
     for my $asset_info (sort { $a->{name} cmp $b->{name} } values %assets) {
         # avoid plain create or we will get unique constraint problems
         # in case ISO_1 and ISO_2 point to the same ISO
+        # note: Not updating the asset size here as doing it in this big transaction
+        #       would lead to deadlocks (see poo#120891).
         my $asset = $assets->find_or_create($asset_info, {for => 'update'});
```

Comment:
As discussed in the unblock, this `FOR UPDATE` should not be here anymore.

Reply:
Handled in https://github.com/os-autoinst/openQA/pull/4981